x-pack/plugins/observability/public/pages/alerts/containers/alerts_table_t_grid/alerts_table_t_grid.tsx

```diff
@@ -201,7 +201,7 @@ function ObservabilityActions({
 
   const actionsMenuItems = useMemo(() => {
     return [
-      ...(casePermissions.crud
+      ...(casePermissions.create
```

Comment:
@cnasikas maybe this should require `read` permissions too? Can't add to an existing case without being able to show the existing cases.